_episodes/01-singularity-gettingstarted.md

```diff
@@ -44,7 +44,9 @@ Working with Singularity containers:
 
 ## What is Singularity?
 
-[Singularity](https://sylabs.io/singularity/) is another container platform. In some ways it appears similar to Docker from a user perspective, but in others, particularly in the system's architecture, it is fundamentally different. These differences mean that Singularity is particularly well-suited to running on distributed, High Performance Computing (HPC) infrastructure, as well as a Linux laptop or desktop! 
+[Singularity](https://sylabs.io/singularity/) is a container platform that allows researchers to easily share their work with others by packaging and deploying their software applications in a portable and reproducible manner. When you download a container created with Singularity, you essentially receive a virtual computer that is preconfigured with all of the necessary software and libraries for a specific research project. This saves you the time and effort of setting up a new computer from scratch, as you can simply run the container and have a virtual environment that is identical to the one used by the original researcher. Container platforms like Singularity provide a convenient and consistent way to access and run research projects, as they allow users to isolate their software environments from the host operating system and run multiple experiments simultaneously.
```

Comment:
Suggesting some revisions to the text based on comments in the PR conversation. Feel free to accept as is, or to use these changes as suggestions and update manually.

```suggestion
[Singularity](https://sylabs.io/singularity/) is a container platform that allows software engineers and researchers to easily share their work with others by packaging and deploying their software applications in a portable and reproducible manner. When you download a Singularity container image, you essentially receive a virtual computer disk that contains all of the necessary software, libraries and configuration to run one or more applications or undertake a particular task, e.g. to support a specific research project. This saves you the time and effort of installing and configuring software on your own system or setting up a new computer from scratch, as you can simply run a Singularity container from the image and have a virtual environment that is identical to the one used by the person who created the image. Container platforms like Singularity provide a convenient and consistent way to access and run software and tools. Singularity is increasingly widely used in the research community for supporting research projects as it allows users to isolate their software environments from the host operating system and can simplify tasks such as running multiple experiments simultaneously.
```